constants/theme.ts

```diff
@@ -1,7 +1,7 @@
 export const appTheme = {
   color: {
-    sunRay: '#FFB319', // primary
-    cheerfulHue: '#FFE194', // secondary
+    cream: '#FFFACC', // primary
+    darkSienna: '#53131E', // secondary
```

Comment:
Here you can add those colors and shadows.

Reply:
Should all the colors be in themes? I presumed that themes are for light, dark etc.? I created another file in constants => colors.ts, but maybe that's redundant?